test/index.js

```diff
@@ -1,95 +1,130 @@
-var tape = require('tape');

```

Comment:
Not talking about indentation, but line endings. The SublimeText EditorConfig plugin should handle this properly though, so I think your commit here actually fixed them. Still, just curious what OS you're using?